### DISCUSSION TITLE
强迫症 希望能调整组件顺序

### DISCUSSION THREAD

**Discussion by KuiPer-LTW** [the1812/Bilibili-Evolved#4900] [功能建议 / Ideas]
如题，新添加的组件只会在最底下，但我就想把它们按功能顺序排列，看着舒服(T▽T)

**Comment by KuiPer-LTW** [comment]
之前都是把组件全部删除然后按顺序重新安装的，但组件越来越多也就越来越麻烦了(T▽T)

**Comment by KuiPer-LTW** [comment]
之前都是把组件全部删除然后按顺序重新安装的，但组件越来越多也就越来越麻烦了(T▽T)

**Comment by the1812** [comment]
只有个按最近使用排序的插件可以用用

**Reply by TiAmoPlus** [reply]
我可以在此讨论上提个类似的功能需求吗？还是需要新开一个讨论呢？
希望功能区的组件，可以按照组件的名字来排序固定下来，而不是按照安装时间顺序去排序。
原因是我会在多个浏览器里安装此脚本，如chrome，edge等，还有可能会卸载重装此脚本，并安装我想要的组件功能。但是由于组件的排序是按照安装顺序排的。我也不太能保证每次安装组件顺序都是一样的。
所以就会出现，chrome里的组件是一个顺序，edge里的组件是另一个顺序了。来回切换使用的时候，总容易按照之前习惯点到错误组件上了。
按照组件名字排序，我觉得是一个稍微算固定顺序的排序。
![PY{`(OFKJZ%H3F}1HM51{XT](https://github.com/user-attachments/assets/81fbf7b0-fc47-45b2-a577-77b1bf66cd02)
![I0_S_H2DDRVWJUTU%XCBQ4Y](https://github.com/user-attachments/assets/86f3dbed-2d2e-4329-b764-7613574bb09e)

**Reply by the1812** [reply]
使用设置里的导入导出应该可以保持顺序一样
